tools/version.sh

```diff
@@ -81,6 +82,14 @@ if [ -z ${VERSION} ] ; then
 
 fi
 
+if [ -z ${EXTRA} ] ; then
+  EXTRA=`git -C ${WD} tag --sort=v:refname | grep -E "nuttx-[0-9]+\.[0-9]+\.[0-9]+" | tail -1 | cut -d'-' -f3`
```

Comment:
maybe, we can extend VERSION to carry the tail part directly.

Reply:
Yes, this is better than duplicating the same `git` command.

Reply:
@xiaoxiang781216 @Ouss4 thanks for the review.
Yeah I agree, and use the remaining of the cut directly in VERSION looks good to me.
I updated the commit